### DISCUSSION TITLE
More control over `ContextMenu`

### DISCUSSION THREAD

**Discussion by elsiniestra** [radix-ui/primitives#2100] [Ideas]
### Overview
I want to make the context menu stick to the trigger component. So no matter where the user clicks on the trigger, the menu must be at the bottom left of the trigger component. Right now, I have no way to do that except by using `#parentComponent [data-radix-popper-content-wrapper] {...some styles}`. Not the best approach especially with the fact that I use Tailwind on the project. It'd be great to add `className` for `ContextMenu.Root`

**Comment by andy-hook** [comment]
@capitalistelite can you provide a visual example of this use case? trying to understand if the context menu is the appropriate primitive for the interaction.

**Reply by elsiniestra** [reply]
<img width="349" alt="image" src="https://user-images.githubusercontent.com/54453888/234150660-1b715ea8-42cf-4778-a896-9eae89f07511.png">
The main idea is that when you click 'b4'/'e5'/whatever other move, you can make changes to the moves node tree
